action.yml

```diff
@@ -51,9 +56,6 @@ runs:
       id: tflint
       shell: bash
       env:
-        # We may want to allow specifying reviewdog and tflint version as
-        # action's input, but let's start with hard coded latest stable version for reviewdog
-        # and latest release for tflint
```

Comment:
Shouldn't you keep the part about reviewdog?

Reply:
I did not think so. I did not see any point in using a specific version of reviewdog. Do you think it would help anybody?

For reviewdog we're always going to be using latest, as it's updated by the newly implemented-depup workflow: https://github.com/reviewdog/action-tflint/pull/33

Reply:
I'm noting that the comment had two parts, one about reviewdog, and one about tflint. Review dog's version is still hard coded.

This PR says it's about a customizable tflint version. Removing a comment that talks about maybe supporting a customizable reviewdog version in a PR for customizable tflint isn't really the right place, it's mixing things.